pgcli/packages/tabulate.py

```diff
@@ -441,10 +441,10 @@ def _align_column(strings, alignment, minwidth=0, has_invisible=True):
 
     """
     if alignment == "right":
-        strings = [s.strip() for s in strings]
+        strings = [s for s in strings]
```

Comment:
Just for readability's sake, I think you should outright delete these lines. Since `strings` is already a list, `[s for s in strings]` just creates an identical list, which is redundant.